Engine_Revit_UI/Convert/Environment/ToBHoM/BuildingElement.cs

```diff
@@ -595,6 +594,36 @@ internal static List<BuildingElement> ToBHoMBuildingElements(this Wall wall, Pul
             return aBuildingElements;
         }
 
+        /***************************************************/
+        /****             Internal methods              ****/
+        /***************************************************/
+
+        private static string GetName(SpatialElement spatialElement)
```

Comment:
Hi @michaldengusiak,

Can we please add todo note here that this should be moved to Query namespace:
```
//TODO: change to proper Query method
private static string GetName(SpatialElement spatialElement)
{
   [...]
}
```